backend/src/libs/packages/config/config.package.ts

```diff
@@ -140,6 +140,20 @@ class Config implements IConfig {
           default: null,
         },
       },
+      GOOGLE_AUTH: {
+        GOOGLE_CLIENT_ID: {
+          doc: 'Google Auth api client id',
+          format: String,
+          env: 'GOOGLE_CLIENT_ID',
+          default: null,
+        },
+        GOOGLE_CLIENT_SECRET: {
```

Comment:
Here we can leave it as it is -- as we already grouped them under `GOOGLE_AUTH`
```suggestion
      GOOGLE_AUTH: {
        CLIENT_ID: {
          doc: 'Google Auth api client id',
          format: String,
          env: 'GOOGLE_CLIENT_ID',
          default: null,
        },
        CLIENT_SECRET: {
```